### PR TITLE
Create OpenStack Keystone Endpoint module

### PR DESCRIPTION
**Pull request by emonty**

### CODE REVIEW

**Comment by bcoca** [comment]
again, is this waiting for something? needs_revision tag indicates PR needs to be revised by author due to comments or its missing fixes

**Comment by gregdek** [comment]
@emonty pushed a bunch of these modules so that they would be visible to others, even though he knew they still needed work.

File: cloud/openstack/os_keystone_endpoint.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  os_keystone_endpoint: >
+    service="image"
+    region="aw2"
+    public_url="https://glance.aw1.bigcompany.com/"
+    internal_url="http://glance-aw1.internal:9292/"
+    admin_url="https://glance.aw1.bigcompany.com/"
+    state="absent"
+    cloud: dguerri
+'''
+
```
Comment (omgjlk) [inline cloud/openstack/os_keystone_endpoint.py:92]: Please document the return data.

**Comment by gregdek** [comment]
@emonty ping for update

**Comment by gregdek** [comment]
@emonty ping again for update

**Comment by emonty** [comment]
On 08/28/2015 10:24 AM, Greg DeKoenigsberg wrote:

> @emonty ping again for update
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/ansible/ansible-modules-core/pull/1593#issuecomment-135838681

Back from vacation, will finish next week

**Comment by gregdek** [comment]
@emonty @shrews @dguerri @juliakreger @j2sol @rcarrillocruz ping

**Comment by gregdek** [comment]
@emonty A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

**Comment by robynbergeron** [comment]
@emonty Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If we don't hear from you within another 14 days, we will close this pull request.

**Comment by Shrews** [comment]
This has been replaced by PR: https://github.com/ansible/ansible-modules-extras/pull/1539

Please close this PR.

**Pull request closed without merging**